lib/stack_master/commands/apply.rb

```diff
@@ -12,7 +12,7 @@ def initialize(config, stack_definition, options = Commander::Command::Options.n
         @stack_definition = stack_definition
         @from_time = Time.now
         @options = options
-        @options.on_failure ||= "ROLLBACK"
+        @options.on_failure ||= @config.stack_defaults['on_failure']
```

Comment:
Yes, that preserves the current functionality while not potentially tripping users up in the future.